### PR TITLE
Adding IPStack

### PR DESCRIPTION
**Pull request by eknowlton**
- Separate database from code
  - [x] Create Parser
  - [x] GeoIP Parser 
- [ ] Add parser tests... maybe? 
- [x] Add IP Stack
  - [x] Add IP Stack Parser

### CODE REVIEW

File: iputil/ipstack/ipstack.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	if res.Connection != nil {
+		if res.Connection.ASN > 0 {
+			parserResponse.ASN = fmt.Sprintf("AS%d", res.Connection.ASN)
+		}
+	}
+
+	return parserResponse, nil
+}
+
+func (ips *IPStack) IsEmpty() bool {
```
Comment (eknowlton) [inline iputil/ipstack/ipstack.go:59]: I guess this is checking to see if location is available, still trying to decode this.

**Comment by eknowlton** [comment]
Some of the fields, noticablely so far ASN, are only available when you pay for IPStack.

**Comment by jacobhaug** [comment]
> Some of the fields, noticablely so far ASN, are only available when you pay for IPStack.

Yep, I'm hoping we can expand our capabilities when using IPStack to include the Time Zone Module, Currency Module, and Connection Module.

**Additional Updates**
- Expand information shown about IP by using the Pro Modules offered by IPStack. I can get you our API key for development if you want.
- We'd like to customize the template as it'll be offered by Level for use by our clients. I'll get Stephen to design something based on the template.

**Comment by jacobhaug** [comment]
**Future Updates**

- Implement a Redis cache, where we store recent lookups to save HTTP request time.
- Ability to restrict API access by token, aka, Level API key.
- Store usage data by token so we can detect abuse/overuse

**Pull request merged**